Examples/Framework/include/ActsExamples/Utilities/detail/ExtractedSimulationProcess.hpp

```diff
@@ -0,0 +1,24 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "ActsExamples/EventData/SimParticle.hpp"
+
+#include <vector>
+
+namespace ActsExamples {
+
+/// Stores the initial properties of a particle, the properties before the
+/// interaction and the particle properties after the interaction
+struct ExtractedSimulationProcess {
+  SimParticle initial;
+  SimParticle before;
+  std::vector<SimParticle> after;
+};
+}  // namespace ActsExamples
```

Comment:
The header should go into `ActsExamples/EventData/` since it is event data shared between multiple modules.

Since you usually store more than one of this object, you should also define a container, e.g.
```
using ExtractedSimulationProcessContainer = std::vector<...>;
 ```
so that all modules use a consistent definition. This is done for all other event data types.

Reply:
done